example_cpp_smart_card_client_app/src/background.js

```diff
@@ -89,6 +89,22 @@ var logger = GSC.Logging.getLogger(
 var naclModule = new GSC.NaclModule(
     'nacl_module.nmf', GSC.NaclModule.Type.PNACL);
 
+// Reload the self app when the NaCl module is disposed of (e.g., due to a
```

Comment:
1) "self app" ?
2) any reason why the // and /**...*/ comment are separated?
3) The listener doesn't actually reload the app, but triggers sth that will cause a reload somewhere else
4) This only happens in release mode (-> mention in comment?)

Reply:
Done with all 1..4. Thanks.